DataFormats/ForwardDetId/interface/ETLDetId.h

```diff
@@ -14,10 +14,10 @@
 
 class ETLDetId : public MTDDetId {
 public:
-  static const uint32_t kETLmoduleOffset = 7;
-  static const uint32_t kETLmoduleMask = 0x1FF;
-  static const uint32_t kETLmodTypeOffset = 5;
-  static const uint32_t kETLmodTypeMask = 0x3;
+  static const uint32_t kETLmoduleOffset = 5;    //7
+  static const uint32_t kETLmoduleMask = 0x7FF;  //0x1FF
```

Comment:
@icosivi @casarsa this change will affect ETLDetLayerGeometryBuilder , but it should be transparent as a result (just less efficient).